cmd/process-agent/main_common.go

```diff
@@ -11,6 +11,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
```

Comment:
Unfortunately [this is likely pulling the dependency](https://github.com/DataDog/datadog-agent/blob/main/cmd/agent/common/common.go#L19) on rtloader again.

Reply:
So that's why the windows builds were failing...

I changed the way I loaded the config so it should be okay now.